README.md

```diff
@@ -109,6 +109,10 @@ You can override any of these default settings by specifying them.
     }
 ```
 
+Note:
+
```

Comment:
When momentjs tries to parse such a date with 4 digits (negative), then it seems to return `undefined`.
Could we add an error to the console in this case when we try to load it from the config?

Reply:
630d687c5824ebc432e62e84ff15ab97c55144a5